payment-request/PaymentRequestUpdateEvent/updateWith-call-immediate-manual.https.html

```diff
@@ -0,0 +1,206 @@
+<!doctype html>
+<meta charset="utf8">
+<link rel="help" href="https://www.w3.org/TR/payment-request/#updatewith()-method">
+<link rel="help" href="https://github.com/w3c/payment-request/pull/591">
+<title>
+  PaymentRequestUpdateEvent.updateWith() needs to be called immediately
+</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+setup({ explicit_done: true, explicit_timeout: true });
+const applePay = Object.freeze({
+  supportedMethods: "https://apple.com/apple-pay",
+  data: {
+    version: 3,
+    merchantIdentifier: "merchant.com.example",
+    countryCode: "US",
+    merchantCapabilities: ["supports3DS"],
+    supportedNetworks: ["visa"],
+  }
+});
+const validMethod = Object.freeze({ supportedMethods: "basic-card" });
```

Comment:
Not sure if it makes a difference, but Chrome has deprecated "basic-card". We only support payment handlers now, such as "https://google.com/pay".

We also deprecated manual install of payment handlers. Only just-in-time install is supported. 

What that means for our tests is that we setup a JIT-installable payment handler at some local URL and then point our tests to it as the `supportedMethods`.

I think it's OK to re-land these tests as-is, but let's keep in mind that `validMethod` may need to change before it works in Chrome correctly.

Reply:
Let's add https://google.com/pay ?

Reply:
OK, let's add `https://google.com/pay`. Our team can follow up later to pass the correct method-specific-data to make Google Pay work in tests, too.

Reply:
that would be great!